src/foam/nanos/approval/ApprovalRequest.js

```diff
@@ -338,6 +338,15 @@
       gridColumns: 3,
       readPermissionRequired: true
     },
+    {
+      class: 'Reference',
+      of: 'foam.nanos.auth.User',
+      name: 'createdFor',
+      includeInDigest: true,
```

Comment:
We need to flag this for testing with Medusa - introducing an includeInDigest property.